db/fixtures/customization_templates.yml

```diff
@@ -275,6 +275,116 @@
   :type: CustomizationTemplateSysprep
   :system: true
 
+- :name: Windows Server 2012 R2
+  :description: Windows Server 2012 R2 Enterprise x64
+  :script: |-
+    <%
+    # Setting Variables
+    debug                   = false
+    time_zone               = evm[:sysprep_timezone].present? ? evm[:sysprep_timezone] : "GMT+1"
+    # Change values if blank
+    evm[:sysprep_computer_name]          = evm[:vm_target_hostname] if evm[:sysprep_computer_name].blank?      # Use vm_target_hostname if hostname in the dialog was blank
```

Comment:
Nitpicking but could you fix the spacing here?  I assume it was lined up before at some point

Reply:
@agrare done, tnx!